### ISSUE TITLE
Workspace: Filter gradient histogram charts or sort by type

### ISSUE THREAD

**Issue by seanswyi** [wandb/wandb#1644]
**Problem or Use Case**
I frequently use `wandb.watch` to observe my model's gradients throughout the training process to make sure training is properly taking place. I thought that it would be convenient if there was a filter function for us to be able to choose which gradients we want to view, considering that there are typically hundreds or thousands of these histogram plots. For example, it would be convenient to be able to select to show only the weights/biases or only the gradients at certain parts of the model.

**Suggested Solution**
A dropdown menu with checkboxes for each component is one idea that I had. For example, if I had a model that looked like the following:

```Python
>>> import torch.nn as nn
>>> class Model(nn.Module):
...     def __init__(self):
...         super(Model, self).__init__()
...         self.linear = nn.Linear(in_features=3, out_features=3)
...         self.softmax = nn.Softmax(dim=-1)
...     def forward(self, x):
...         linear_output = self.linear(x)
...         return self.softmax(linear_output)
>>> model = Model()
>>> print(model)
Model (
    (linear): Linear(in_features=3, out_features=3, bias=True)
    (softmax): Softmax(dim=-1)
)
```

then maybe there could be something like

- [ ] All
- [ ] Model
  - [ ] linear
  - [ ] softmax

**Alternatives**
Alternatives would be to just sort according to `weight` or `bias`, since that in and of itself would also be convenient.

**Comment by cvphelps** [comment]
Hi @seanswyi good feature request. Have you tried the search box at the top of the run page workspace? This might let you get the filtering you're looking for if you type "weight" or "bias" in there

**Comment by seanswyi** [comment]
Hey @cvphelps ! Yes, I've tried using the search feature as well. Although it's usually sufficient for whatever I want to achieve, it just popped into my mind that it may be more convenient to have this filtering feature, as I'm aware that it seems if you use the search feature then your charts would disappear unless they contain the information you're trying to search for.

**Comment by sydholl** [comment]
We are closing this issue due to inactivity. If you feel that it merits additional review, please provide a reproduction and add a comment explaining why. Thank you.
